mapwms.cpp

```diff
@@ -3770,7 +3770,7 @@ int msWMSGetMap(mapObj *map, int nVersion, char **names, char **values, int nume
   } else {
 
     /* intercept requests for Mapbox vector tiles */
-    if(!strcmp(MS_IMAGE_MIME_TYPE(map->outputformat), "application/x-protobuf")) {
+    if(!strcmp(MS_IMAGE_MIME_TYPE(map->outputformat), "application/vnd.mapbox-vector-tile")) {
```

Comment:
you could accept both outputformats here for backward compatibility

Reply:
I've changed it now so it should be more backwards compatible than before - application/x-protobuf can still be requested successfully. It is also now still advertised in the GetCapabilities as a GetMap format.

The only backwards incompatibility I can think of now is that an output format defined like https://github.com/sdlime/mvt-demo/blob/master/demo.map#L24 will use correct mime type now instead of the old one.